openfasoc/generators/glayout/glayout/flow/blocks/Divide_by_two_SystemsGenesys/divide_by_two.py

```diff
@@ -0,0 +1,30 @@
+from gdsfactory import Component
+from glayout.flow.routing.straight_route import straight_route
+from glayout.flow.routing.c_route import c_route
+from latch_design import create_nmos_latch_layout
+
+def create_divide_by_two_circuit(pdk):
+    divider = Component("divide_by_two")
```

Comment:
You can generate the spice netlist for this once the latch netlist is generated. Refer to https://github.com/idea-fasoc/OpenFASOC/blob/e7f461123dd9e95216ac9cd59f057c8922fe47de/openfasoc/generators/glayout/glayout/flow/blocks/composite/opamp/opamp.py#L26